### DISCUSSION TITLE
Customizing quick entry site

### DISCUSSION THREAD

**Discussion by GarryBlue** [kimai/kimai#4713] [Questions & Answers]
Hi there
We'd like to provide costumers the opportunity to bulk-add time entries for a single day. 
I'm seeking guidance on customizing the quick entry site for single-day time entry with an additional fixedRate field. 

I tried to make changes to the quick-entry index.html.twig. 
I'm using Visual Studio Code with the Symfony for VScode extension. Still it is quite unhandy to work with.
Any insights or recommendations for tools/plugins to manage dependencies would be greatly appreciated. 

Thank you

**Comment by salvamaineelca** [comment]
For the record, a way to modify it is to create a QuickEntryControllerDecorator (following Symfony standards) and then you can modify both the view and the behaviour. The problem is that then you need to duplicate the original code / view (and apply the changes included in future Kimai versions)
